src/com_tjvendors/admin/models/forms/vendor.xml

```diff
@@ -1,27 +1,243 @@
-		<field name="vendor_logo" type="file" id="vendor_logo" accept="image/jpeg,image/png,image/jpg" label="COM_TJVENDORS_FORM_LBL_VENDOR_PROFILE_AVATAR" description="COM_TJVENDORS_FORM_DESC_VENDOR_PROFILE_AVATAR" hint="COM_TJVENDORS_FORM_LBL_VENDOR_PROFILE_AVATAR" filter="" />
-		<field name="checked_out_time" default="0000-00-00 00:00:00" type="hidden" filter="" />
-		<field name="state" default="1" type="list" label="JSTATUS" description="JFIELD_PUBLISHED_DESC" class="inputbox" size="1">
+		<field
+			name="vendor_id"
+			type="text"
+			default="0"
+			label="COM_TJVENDORS_FORM_LBL_VENDOR_ID"
+			readonly="true"
+			class="readonly "
+			description="JGLOBAL_FIELD_ID_DESC" />
+
+		<field
+			name="user_id"
+			type="user"
+			label="COM_TJVENDORS_FORM_LBL_VENDOR_USER_ID"
+			description="COM_TJVENDORS_FORM_DESC_VENDOR_USER_ID"
+			hint="COM_TJVENDORS_FORM_LBL_VENDOR_USER_ID"
+			menu_published="0"
+			filter=""
+			directory_stripext="true"
+			directory_hidenode="true"
+			directory_hidedefault="true"
+			heading="h4"
+			close="true"
+			option_on="Yes"
+			option_off="Yes"
+			toggle_default_value="0" />
+
+		<field
+			name="vendor_client"
+			type="text"
+			label="COM_TJVENDORS_FORM_LBL_VENDOR_CLIENT"
+			description="COM_TJVENDORS_FORM_DESC_VENDOR_CLIENT"
+			hint="COM_TJVENDORS_FORM_LBL_VENDOR_CLIENT"
+			menu_published="0"
+			filter=""
+			directory_stripext="true"
+			directory_hidenode="true"
+			directory_hidedefault="true"
+			heading="h4"
+			close="true"
+			option_on="Yes"
+			option_off="Yes"
+			toggle_default_value="0" />
+
+		<field
+			name="version_note"
+			type="text"
+			label="JGLOBAL_FIELD_VERSION_NOTE_LABEL"
+			description="JGLOBAL_FIELD_VERSION_NOTE_DESC"
+			class="inputbox"
+			size="45"
+			labelclass="control-label" />
+
+		<field
+			name="vendor_title"
+			required="true"
+			type="text"
+			size="40"
+			class="inputbox"
+			label="COM_TJVENDORS_FORM_LBL_VENDOR_TITLE"
+			description="COM_TJVENDORS_FORM_DESC_VENDOR_TITLE"
+			hint="COM_TJVENDORS_FORM_LBL_VENDOR_TITLE"
+			filter=""/>
+
+		<field
+			name="alias"
+			type="text"
+			label="COM_TJVENDORS_FORM_LBL_ALIAS"
+			description="COM_TJVENDORS_FORM_DESC_ALIAS"
+			filter="" />
+
+		<field
+			name="vendor_description"
+			type="textarea"
+			class="inputbox"
+			label="COM_TJVENDORS_FORM_LBL_VENDOR_PROFILE_DESCRIPTION"
+			description="COM_TJVENDORS_FORM_DESC_VENDOR_PROFILE_DESCRIPTION"
+			hint="COM_TJVENDORS_FORM_LBL_VENDOR_PROFILE_DESCRIPTION"
+			rows="10"
+			cols="5"
+			filter=""/>
+
+		<field
+			name="vendor_logo"
+			type="file"
+			id="vendor_logo"
+			accept="image/jpeg,image/png,image/jpg"
+			label="COM_TJVENDORS_FORM_LBL_VENDOR_PROFILE_AVATAR"
+			description="COM_TJVENDORS_FORM_DESC_VENDOR_PROFILE_AVATAR"
+			hint="COM_TJVENDORS_FORM_LBL_VENDOR_PROFILE_AVATAR"
+			filter="" />
+
+		<field
+			name="first_name"
```

Comment:
@deepgawade89 We have vendor title field. Don't add separate firstname lastname field.

Reply:
Updated